src/components/ClusterOverview/GPUStatus.tsx

```diff
@@ -96,12 +96,12 @@ const getTemperatureHealth: SubsystemHealthHandler = (responses) => {
 
   temperatures.sort((a, b) => b - a);
 
-  if (temperatures[0] > 70) {
+  if (temperatures[0] > 110) {
```

Comment:
Since we can not get these dynamically, it would be good to keep those threshold colocated (as exported constants from a single file)